src/bootstrap/lib.rs

```diff
@@ -867,6 +871,12 @@ impl Build {
         return base
     }
 
+    /// Returns if this target should statically link the C runtime, if specified
+    fn crt_static(&self, target: &str) -> Option<bool> {
+        self.config.target_config.get(target)
+            .and_then(|t| t.crt_static)
+    }
```

Comment:
Since +crt-static was previously unconditional for msvc, I left it that way when moving the check over.